<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formWidget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>391</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_kepengurusan">
      <property name="text">
       <string>ID Kepengurusan</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_id_kepengurusan"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_nama_pengurus">
      <property name="text">
       <string>Nama Pengurus</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_nama_pengurus"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_partai_politik_yang_diurus">
      <property name="text">
       <string>Partai Politik yang Di Urus</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit_partai_politik_yang_diurus"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edit_kewenangan"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbl_kewenangan">
      <property name="text">
       <string>Kewenangan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_simpan">
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
